src/Backend/Version.ts

```diff
@@ -16,19 +16,27 @@
 
 class Version {
   major: number;
-  minor: number;
-  patch: number;
+  minor: number|undefined;
+  patch: number|undefined;
   option: string;
 
-  constructor(major: number = 0, minor: number = 0, patch: number = 0, option: string = '') {
+  constructor(
+      major: number = 0, minor: number|undefined, patch: number|undefined, option: string = '') {
```

Comment:
Some of parameters have default value but others are not.
How about making them consistent? :)

Reply:
This change supports for `1.0-beta` version. Before this changes were applied, the version was described as `1.0.0-beta`. Because there is no way to express empty patch version. For this one, I proposed the `undefined` variable for `minor` and `patch`.

Reply:
Upon @llFreetimell's comment, I tried some calls, it seems somewhat confusing.
```
Version();  -> compile error
Version(1);   -> compile error
Version(1, 2);   -> compile error
Version(undefined, undefined); -> compile error
foo(1, 2, 3);   -> OK
foo(1, 2, 'abc');   -> compile error
```

How about changing `major: number = 0` => `major: number`? It seems that `= 0` does not working.